### DISCUSSION TITLE
Congrats! Your request for @Pikamon46 was approved and your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by Pikamon46** [community/community#69576] [GitHub Education] [answered]
### Hi everyone,

So its been 2 days since I was approved so how do I get the benefits. How many more days do I have to wait?

2nd Q: Do I get everything in the student Developer pack or do I get to choose only one ?

**Comment by mantasu** [comment]
Hey, I had the same issue, you just have to wait for a few more days (less than a week) as there will be some time before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, you can just open a [ticket](https://support.github.com/request/education).

You get all the benefits in the Student Dev Pack

**Reply by Pikamon46** [reply]
ok I will wait

**Comment by santoshvandari** [comment, marked as answer]
Hi "Pikamon46,"

Congratulations on getting approved for GitHub Education benefits! I understand you're excited to start using them. Here are the answers to your questions:

Accessing Benefits: After your request has been approved, it may take a little bit of time for the benefits to become available to you. The exact timing can vary, but you should receive information about how to access your benefits via email or through your GitHub account. Check your email, including your spam folder, for any communications from GitHub Education. If you haven't received instructions after a reasonable amount of time, consider contacting GitHub Support for assistance.

Student Developer Pack: The GitHub Student Developer Pack includes a variety of valuable tools and resources from different companies. Typically, you are eligible to access all of the items listed in the pack. You don't have to choose just one; you can access multiple tools and services that are part of the pack. Each item within the pack may have its own instructions for activation, so be sure to follow those steps to take full advantage of the benefits.

In summary, please keep an eye on your email and GitHub account for instructions on accessing your benefits. You should have access to multiple items from the Student Developer Pack, so explore the offerings and use the most relevant tools to your projects and learning goals.

If you have any further questions or need more specific guidance, don't hesitate to ask. Enjoy your GitHub Education benefits!

Best regards,

**Reply by Pikamon46** [reply]
I got it today thanks
